doc/user-guide-v1beta2.adoc

```diff
@@ -67,15 +67,17 @@ Each `OpenLibertyApplication` CR must specify `.spec.applicationImage` field. Sp
 | `pullSecret` | If using a registry that requires authentication, the name of the secret containing credentials.
```

Comment:
We can skip updating v1beta2 documentation, since the new features/config don't work (not expected to work either).

Reply:
Thank you for catching that, I was able to remove more to prevent configuring `.service.sessionAffinity` in `v1beta2` CRs but will still need to keep the no-op for implementing BaseComponent.